distribution/lib/Standard/Base/0.0.0-dev/src/Data/Range.enso

```diff
@@ -18,7 +53,9 @@ type Range
 
             0.up_to 100 . length
     length : Number
-    length = this.end - this.start
+    length = case this.last of
```

Comment:
why not use `is_empty`

Reply:
Because I use the `last` argument later anyway - so IMO it is better to get the last argument and check if it is not nothing rather than check is_empty and then rely on an implicit nowhere-enforced invariant that `is_empty=False ==> last != Nothing`. Also `last` itself has the `is_empty` check, so we avoid checking it twice.

Reply:
ok fair enough.